HollowEngine/BulletHell/src/Systems/AttackSystem.cpp

```diff
@@ -28,6 +29,15 @@ namespace BulletHell
 			"IsFired", &Attack::mIsFired
 			);
 
+		// Temporary, find a way to get game component in engine
+		lua.new_usertype<Hollow::GameObject>("GameObject",
+			sol::constructors<Hollow::GameObject()>(),
+			"GetBody", &Hollow::GameObject::GetComponent<Hollow::Body>,
+			"GetTransform", &Hollow::GameObject::GetComponent<Hollow::Transform>,
+			"GetMaterial", &Hollow::GameObject::GetComponent<Hollow::Material>,
+			"GetAttack", &Hollow::GameObject::GetComponent<Attack>
+			);
+
```

Comment:
This should work instead.. https://github.com/ThePhD/sol2/blob/develop/examples/source/usertype_advanced.cpp#L81

Reply:
Fixed